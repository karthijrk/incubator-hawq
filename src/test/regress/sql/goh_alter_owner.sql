CREATE ROLE super SUPERUSER;
SET SESSION AUTHORIZATION super;
CREATE ROLE u1;
CREATE DATABASE u1;
CREATE SCHEMA u1;
select nspname, rolname from pg_namespace n, pg_authid a where n.nspowner = a.oid and nspname = 'u1';
select datname, rolname from pg_database d, pg_authid a where d.datdba = a.oid and datname = 'u1';
alter database u1 owner to u1;
alter schema u1 owner to u1;
select nspname, rolname from pg_namespace n, pg_authid a where n.nspowner = a.oid and nspname = 'u1';
select datname, rolname from pg_database d, pg_authid a where d.datdba = a.oid and datname = 'u1';
RESET SESSION AUTHORIZATION;
DROP DATABASE u1;
DROP SCHEMA u1;
DROP ROLE u1;
DROP ROLE super;
