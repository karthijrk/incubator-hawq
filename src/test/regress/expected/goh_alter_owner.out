CREATE ROLE super SUPERUSER;
SET SESSION AUTHORIZATION super;
CREATE ROLE u1;
NOTICE:  resource queue required -- using default resource queue "pg_default"
CREATE DATABASE u1;
CREATE SCHEMA u1;
select nspname, rolname from pg_namespace n, pg_authid a where n.nspowner = a.oid and nspname = 'u1';
 nspname | rolname 
---------+---------
 u1      | super
(1 row)

select datname, rolname from pg_database d, pg_authid a where d.datdba = a.oid and datname = 'u1';
 datname | rolname 
---------+---------
 u1      | super
(1 row)

alter database u1 owner to u1;
ERROR:  Cannot support alter database owner statement yet
alter schema u1 owner to u1;
ERROR:  Cannot support alter schema owner statement yet
select nspname, rolname from pg_namespace n, pg_authid a where n.nspowner = a.oid and nspname = 'u1';
 nspname | rolname 
---------+---------
 u1      | super
(1 row)

select datname, rolname from pg_database d, pg_authid a where d.datdba = a.oid and datname = 'u1';
 datname | rolname 
---------+---------
 u1      | super
(1 row)

RESET SESSION AUTHORIZATION;
DROP DATABASE u1;
DROP SCHEMA u1;
DROP ROLE u1;
DROP ROLE super;
